packages/tools/utils/localenv.js

```diff
@@ -0,0 +1,256 @@
+      'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoicG9zdGdyZXMifQ.oM0SXF31Vs1nfwCaDxjlczE237KcNKhTpKEYxMX-jEU',
+  },
+  {
+    name: 'PINATA_JWT',
+    desc:
+      'Pinata API token. Go to  https://app.pinata.cloud/keys and make a key with all Pinning API permissions.',
+  },
+  {
+    name: 'DAG_CARGO_HOST',
+    desc:
+      'hostname / ip for dag-cargo database (read-only replica). Get from team 1Password if working on PL-managed instance.',
+  },
+  {
+    name: 'DAG_CARGO_DATABASE',
+    desc:
+      'hostname / ip for dag-cargo database (read-only replica). Get from team 1Password if working on PL-managed instance.',
+  },
+  {
+    name: 'DAG_CARGO_USER',
+    desc:
+      'username for dag-cargo database (read-only replica). Get from team 1Password if working on PL-managed instance.',
+  },
+  {
+    name: 'DAG_CARGO_PASSWORD',
+    desc:
+      'password for dag-cargo database (read-only replica). Get from team 1Password if working on PL-managed instance.',
+  },
+]
+
+const REQUIRED_WRANGLER_SECRETS = [
+  {
+    name: 'DATABASE_TOKEN',
+    desc:
+      "Signed JWT for local db API. Use the default if you're using the devcontainer / docker-compose setup.",
+    defaultValue:
+      'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoicG9zdGdyZXMifQ.oM0SXF31Vs1nfwCaDxjlczE237KcNKhTpKEYxMX-jEU',
+  },
+  {
+    name: 'MAGIC_SECRET_KEY',
+    desc:
+      'Secret API key for magic.link. Sign up for an account at https://magic.link and make a new project, then use the secret key.',
+  },
+  {
+    name: 'PINATA_JWT',
+    desc:
+      'Pinata API token. Go to  https://app.pinata.cloud/keys and make a key with all Pinning API permissions.',
+  },
+  {
+    name: 'SALT',
+    desc: 'Random salt. See packages/api/README.md',
+  },
+  {
+    name: 'SENTRY_DSN',
+    desc:
+      'Sentry DSN. Sign up for an account at https://sentry.io, make a new app, and copy the DSN.',
+  },
+]
+
+class LocalEnv {
+  static async lint() {
+    const envOk = await this.lintEnvFile()
+    let wranglerOk = true
+    let loggedIn = await this.checkWranglerIsAuthenticated()
+    if (!loggedIn) {
+      loggedIn = await this.tryWranglerLogin()
+    }
+
+    if (loggedIn) {
+      wranglerOk = await this.checkWranglerToml()
+    } else {
+      console.warn(
+        'cloudflare wrangler not authenticated. skipping validation of wrangler config.'
+      )
+    }
+
+    if (envOk && wranglerOk) {
+      console.log('Everything looks okay!')
+      console.log(
+        'If this is a new dev env, run the following to setup the db:'
+      )
+      console.log('node ./packages/api/scripts/cli.js db-sql --cargo --testing')
+    } else {
+      console.error(
+        'Your config seems to have errors. Please address the issues printed above and try again'
+      )
+    }
+  }
+
+  static async checkWranglerIsAuthenticated() {
+    console.log('verifying that wrangler is authenticated...')
+    try {
+      childProcess.execSync('wrangler whoami')
```

Comment:
This is returning the user `node` for me. Then complains about not being able to find `/home/node/.wrangler/config/default.toml` :(